client/src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+import Button from "../UIElements/Button";
+import "./Navbar.scss";
+import { authStore } from "../../stores/AuthStore";
+import { observer } from "mobx-react-lite";
+
+const Navbar = observer(() => {
+  const handleLogout = () => {
+    authStore.logout();
+  };
+  const navbarClass = authStore.isLogged ? 'navbar logged' : 'navbar';
+  return (
+    <div className={navbarClass}>
+      {authStore.isLogged && (
+        <Link to="/login">
+          <Button inverse onClick={handleLogout}>התנתק</Button>
```

Comment:
You can drop handleLogout funciton, its redundant here.

```suggestion
          <Button inverse onClick={authStore.logout}>התנתק</Button>
```